src/views/Order.vue

```diff
@@ -290,28 +293,53 @@ export default {
       status: "Approved" // default value
     }
   },
-  methods: {
-    changeStatus (ev) {
+  computed: {
+    ...mapGetters({
+      order: 'order/getCurrentOrder',
+      getProduct: 'product/getProduct',
+      getProductStock: 'stock/getProductStock',
+      getShipmentMethodDescription: 'util/getShipmentMethodDesc'
```

Comment:
Either the getter should return description directly or name should improved

Reply:
Changed method name to `getShipmentMethod` and return the shipment desc from the getter only.